sdk/core/azure-core-http-netty/src/main/java/com/azure/core/http/netty/implementation/HttpProxyHandler.java

```diff
@@ -0,0 +1,331 @@
+/*
+ * Copyright 2014 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+/*
+ * Portions Copyright (c) Microsoft Corporation
+ */
```

Comment:
Would like to get @JonathanGiles opinion on this copyright section.

Reply:
@anuchandy Could you also weigh in on this as you've been investigating this area recently

Reply:
I believe this was the advice that @anuchandy received, so I will defer to him. It seems reasonable to me. I would expect that the THIRD_PARTY_NOTICES file is also updated.

Reply:
Updated the licensing file to include the copyright above.